src/test/java/com/linecorp/armeria/client/http/HttpClientIntegrationTest.java

```diff
@@ -140,6 +148,48 @@ public static void destroy() throws Exception {
         });
     }
 
+    /**
+     * Tests if the {@link HttpClient} can send a minimum possible request.
```

Comment:
It's a PR with two commits with two fixes actually. I can split this into two PRs but it's not that big.